web/src/components/tasks/anytime-action/tax-clearance-certificate/AnytimeActionTaxClearanceCertificateElementAlert.tsx

```diff
@@ -0,0 +1,71 @@
+import { Alert } from "@/components/njwds-extended/Alert";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { getProfileConfig } from "@/lib/domain-logic/getProfileConfig";
+import { ProfileContentField } from "@/lib/types/types";
+import { ReactElement } from "react";
+
+interface Props {
+  fieldErrors: string[];
+}
+
+export const AnytimeActionTaxClearanceCertificateElementAlert = (props: Props): ReactElement | null => {
+  const { Config } = useConfig();
+
+  const displayTaxClearanceErrorAlert = (): boolean => props.fieldErrors.length > 0;
+
+  const customOrder = [
+    "requestingAgencyId",
+    "businessName",
+    "entityId",
+    "addressLine1",
+    "addressLine2",
+    "addressCity",
+    "addressState",
+    "addressZipCode",
+    "taxId",
+    "taxPin",
+  ];
+  const filteredOrder = customOrder.filter((item) => props.fieldErrors.includes(item));
+  props.fieldErrors.sort((a, b) => filteredOrder.indexOf(a) - filteredOrder.indexOf(b));
+
+  const getLabel = (field: string): string => {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-expect-error
+    const formationLabel = Config.formation.fields[field]?.label;
+
+    let requestingAgencyLabel = "";
+
+    if (field === "requestingAgencyId") {
+      requestingAgencyLabel = "Requesting Agency";
```

Comment:
this value should exist in the cms somewhere, we don't hard code text that is rendered in the UI because it is not editable by content.